lib/ios/AirMaps/AIRMapManager.m

```diff
@@ -716,9 +725,6 @@ - (void)mapViewWillStartRenderingMap:(AIRMap *)mapView
 - (void)mapViewDidFinishRenderingMap:(AIRMap *)mapView fullyRendered:(BOOL)fullyRendered
 {
     [mapView finishLoading];
-    [mapView cacheViewIfNeeded];
```

Comment:
This was also an issue: `cacheViewIfNeeded` is called on `finishLoading` also, which is a tremendous overhead that is not necessary.
As caching an image is a CPU and RAM consuming job.

Reply:
Hi, I'm sorry to say but this commit is quite flawed. It creates a global variable called `didCallOnMapReady` which once set to YES is never reset to NO. This breaks `onMapReady` quite badly, causing it now to only fire on the first rendered MapView and after that never again (not when MapView is unmounted/mounted, or for any other MapViews).

@danielgindi @christopherdro

Reply:
Fixed by PR #1853